package.json

```diff
@@ -24,11 +24,16 @@
   },
   "scripts": {
     "build": "rollup -c",
-    "test": "node --experimental-modules node_modules/c8/bin/c8 --reporter=html --reporter=lcov --reporter=text --check-coverage node --experimental-modules node_modules/mocha/bin/mocha",
+    "test:nocoverage": "mocha --colors",
+    "test": "c8 --reporter=html --reporter=lcov --reporter=text --check-coverage npm run test:nocoverage",
+    "test:experimental:nocoverage": "mocha --experimental-modules --colors",
+    "test:experimental": "c8 --reporter=html --reporter=lcov --reporter=text --check-coverage npm run test:experimental:nocoverage",
+    "test:esm:nocoverage": "mocha -r esm --colors",
+    "test:esm": "c8 --reporter=html --reporter=lcov --reporter=text --check-coverage npm run test:esm:nocoverage",
+    "test:cjs-build": "node ./test/commonjs/test-artifact.js",
     "coverage": "c8 report --reporter=text-lcov | coveralls",
     "test-types": "tsd",
-    "lint": "xo",
-    "prepublishOnly": "node ./test/commonjs/test-artifact.js"
```

Comment:
Why we don't want to test this anymore?

Reply:
@tinovyatkin We do want it. It's just been renamed to `test:cjs-build` since it's not just a pre-publish step.

Reply:
But should we still call this from prepublishOnly? Why we are completely removing prepublish check?

Reply:
@tinovyatkin Ah, you are quite right. I've fixed this.